ugvc/mrd/featuremap_utils.py

```diff
@@ -211,12 +217,24 @@ def __init__(
         self.CYCLE_SKIP_FLAG = FeatureMapFields.IS_CYCLE_SKIP.value
         self.PREV_N_BP = f"prev_{self.motif_length_to_annotate}bp"
         self.NEXT_N_BP = f"next_{self.motif_length_to_annotate}bp"
+        self.NEXT_1 = "next_1sbp"
+        self.NEXT_2 = "next_2sbp"
+        self.NEXT_3 = "next_3sbp"
+        self.PREV_1 = "prev_1sbp"
+        self.PREV_2 = "prev_2sbp"
+        self.PREV_3 = "prev_3sbp"
         self.info_fields_to_add = [
             self.TRINUC_CONTEXT_WITH_ALT,
             self.HMER_CONTEXT_REF,
             self.HMER_CONTEXT_ALT,
             self.PREV_N_BP,
             self.NEXT_N_BP,
+            self.NEXT_1,
```

Comment:
if we have a list of features, we could just join the lists here:
info_fields_to_add = [F1, F2, ... ] + ref_bases_feature_names

Reply:
done